app/packages/acter_trigger_auto_complete/lib/src/acter_text_controller.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+
+/// Extended [TextEditingController] which takes up trigger styles as optional.
+/// Styles the trigger inputs based on map for mentions, hashtags or emojis etc.
+/// If provided empty, will
+class ActerTextController extends TextEditingController {
+  ActerTextController({String? text, this.triggerStyles}) : super(text: text);
```

Comment:
I think calling this `ActerTextController` is a bit weird. either call it `ActerTriggerAutoCompleteTextController` or just `TriggerAutocompleteTextController`?!?

Reply:
First one sounds better with more meaningful to plugin. Will update it!